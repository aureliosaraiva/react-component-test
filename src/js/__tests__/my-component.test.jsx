/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import MyComponent from '../my-component';

jest.unmock('../my-component');

describe('MyComponent', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <MyComponent />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
