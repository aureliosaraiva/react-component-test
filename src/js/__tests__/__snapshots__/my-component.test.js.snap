exports[`MyComponent renders correctly 1`] = `
<div
  className="my-class">
  My external component
</div>
`;
